internals/secrethub/migrate_config_templates.go

```diff
@@ -48,7 +47,7 @@ func (cmd *MigrateConfigTemplatesCommand) Run() error {
 	return nil
 }
 
-func migrateTemplateTags(inFile io.Reader, outFile io.Writer, mapping referenceMapping, formatString string) ([]string, error) {
+func migrateTemplateTags(inFile io.Reader, outFile string, mapping referenceMapping, formatString string) ([]string, error) {
```

Comment:
I'd say let's take the same approach here with the signature as the `migrateReferences` function and just work with the file contents string, because mixing filepaths with io readers is a bit confusing.

Reply:
Adjusted the signature accordingly.